pkg/logger/logger.go

```diff
@@ -67,6 +67,16 @@ func GetLumberHook() func(e zapcore.Entry) error {
 	return nil
 }
 
+// put error log into a channel for ECS when higher than error level
```

Comment:
You don't need to create a new func, you can just add a new method to `ECSErrorLogger` and move it to `ecs_error_log.go`.

```go
func (e *ECSErrorLogger) GetErrrorHook() ....
````

Reply:
sure